src/styles/utils/index.ts

```diff
@@ -1575,7 +1575,18 @@ const createStyleUtils = (theme: ThemeColors, styles: ThemeStyles) => ({
      * also have an impact on the width of the character, but as long as there's only one font-family and one font-size,
      * this method will produce reliable results.
      */
-    getCharacterPadding: (prefix: string): number => prefix.length * 10,
+    getCharacterPadding: (prefix: string): number => {
+        let padding = 0;
+        prefix.split('').forEach((char) => {
+            if (char === char.toUpperCase()) {
+                padding += 10;
+            } else {
+                padding += 8;
+            }
```

Comment:
Can you please explain this block of code please? What's the reason for `8` value?

Reply:
@youssef-lr is there any case that we mix uppercase and lower case with our prefix?

Reply:
>is there any case that we mix uppercase and lower case with our prefix?

Yes, the symbol for AED for example is `Dhs`

Reply:
I don't think to worry about this for now, we could revisit it later. The input with a prefix for now is only used for the RoomInput component, and the new money request amount in the splits page.

Reply:
So I think let's just make sure the room input name looks as before (10px width). If any changes are needed for the amount used in the split page, we can work on those changes in my PR.